packages/constants/package.json

```diff
@@ -1,6 +1,12 @@
 {
   "name": "@masknet/constants",
   "private": true,
+  "exports": {
+    ".": "./dist/src/index.js",
+    "./utils": "./dist/src/utils.js"
```

Comment:
i think no need expose utils

Reply:
Please make sure it can be imported by TypeScript without type error when using `package export` in package.json.